### PR TITLE
world save fix

### PR DESCRIPTION
**Pull request by Thomas-Harit**

### CODE REVIEW

File: cubic-server/default/DefaultWorld.cpp [hunk 1]
```diff
@@ -26,4 +30,78 @@ void DefaultWorld::initialize()
...
+                    new nbt::String("Name", "1.19"),
+                    new nbt::String("Series", "main"),
+                    new nbt::Byte("Snapshot", 0)
+            }),
+            new nbt::IntArray("WanderingTraderId", {0, 0, 0, 0}),
+            new nbt::Int("WanderingTraderSpawnChance", 10),
+            new nbt::Int("WanderingTraderSpawnDelay", 100000),
+            new nbt::Byte("WasModded", 0)
+    });
+    std::vector<uint8_t> buffer = _data->serialize();
```
Comment (huntears) [inline cubic-server/default/DefaultWorld.cpp:102]: You need to destroy the nbt object after serializing it, else it will leak

**Review by huntears — CHANGES_REQUESTED** [review]

**Comment by STMiki** [comment]
This pr exists 🙃

**Comment by huntears** [comment]
> This pr exists upside_down_face

Overall the PR is still fine in today's code, it can still be built upon I would say

**Comment by huntears** [comment]
Currently working on it in #256

**Pull request closed without merging**